nginx/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# a minimal Nginx container including containerbuddy and a simple virtulhost config
+FROM nginx:latest
```

Comment:
> The Containerbuddy configuration is embedded in the container build [...] the Cloudflare container that results from this is a demo [...] and doesn't stand alone.

Agreed, though would it be easier for people to clone the repo and modify the right bits if we structured it differently?

Ideally, people would feel like they have to make only some very small changes to get going, but the structure here makes it feel like those changes are larger than they really are.